### PR TITLE
fixed ncurses bug in Makefile when specifying TIC_PATH in install (Issue #806)

### PR DESCRIPTION
**Pull request by eruffaldi**

### CODE REVIEW

File: src/ncurses.mk [hunk 1]
```diff
@@ -50,6 +50,6 @@ define $(PKG)_BUILD
             --with-normal    --without-shared --with-static, \
             --without-normal --without-static --with-shared)
     $(MAKE) -C '$(1)' -j '$(JOBS)'
-    $(MAKE) -C '$(1)' -j 1 install TIC_PATH='$(1).native/progs/tic'
+    $(MAKE) -C '$(1)' -j 1 install -s TIC_PATH='$(1).native/progs/tic'
```
Comment (TimothyGu) [inline src/ncurses.mk]: Does this work:

```make
TIC_PATH='$(1).native/progs/tic' $(MAKE) -C '$(1)' -j 1 install
```

Comment (eruffaldi) [inline src/ncurses.mk]: Yes. This is more clean

Comment (tonytheodore) [inline src/ncurses.mk]: I can see the same issue on OSX 10.10 and while both of these allow the build to continue, I suspect they only do so if `tic` is already available.

**Comment by tonytheodore** [comment]
Fixed in https://github.com/mxe/mxe/commit/a6aeeb03cfc0317233cd63621a62775bf7785341

**Pull request closed without merging**